custom/ewsghana/api.py

```diff
@@ -658,6 +658,9 @@ def web_user_sync(self, ews_webuser):
             if self.domain not in user.get_domains():
                 user.add_domain_membership(self.domain, location_id=location_id)
 
+            # We are migrating only active users
+            user.is_active = True
```

Comment:
No but we still need to migrate data to final domain so it's not a huge problem.